math/src/polismath/darwin/core.clj

```diff
@@ -56,25 +53,6 @@
   (update-export-status darwin params {:status status :params params}))
 
 
-;; We use AWS to store conversation exports whcih took a long time to compute.
-;; These exports are set to expire automatically.
-;; The number of days before expiry should be stored in env variable :export-expiry-days.
-;; This value is used to compute the expiry of the exports postgres blob as well.
-
-(defn aws-cred [darwin]
-  (-> darwin :config :aws))
-
-(defn full-aws-path
-  [darwin filename]
-  (str (-> darwin :config :math-env-string) "/" filename))
-
-(defn upload-to-aws
-  [{:as darwin :keys [s3-client]} filename]
-  (aws/invoke s3-client
-              {:op :PutObject
-               :request {:Bucket "polis-datadump"
-                         :Body (io/input-stream (io/file filename))
-                         :Key (full-aws-path darwin filename)}}))
```

Comment:
I forgot about this... we may actually want to keep the Cognitect AWS api dependency. That _is_ reasonably up to date and well maintained by the Cognitect folks (again, stewards of Clojure itself). We don't depend any more on Amazonica so should definitely remove that one, though you may find it was only actually still in the `project.clj` and so not in the classpath any longer.

This part of the system (automatic exports) is actually broken right now, and could potentially be fixed, but we're considering the direction we want to take exports. Can discuss further.

Reply:
In any case, if we decide not to keep the automated export system, this entire namespace can be deleted, since it's only purpose is triggering the exports from admin client and putting them on S3. For command line exports, this namespace is unecessary.

Reply:
Just confirmed, we _do_ want to revive this feature, so please leave this in for now. Should be enough to just make sure the Cognitect AWS lib is up to date.

Reply:
OK - I have restored this. The AWS libs are good according to Snyk so happy to re-instate.